spring-core/src/main/java/org/springframework/util/StreamUtils.java

```diff
@@ -83,7 +83,7 @@ public static String copyToString(@Nullable InputStream in, Charset charset) thr
 			return "";
 		}
 
-		StringBuilder out = new StringBuilder(BUFFER_SIZE);
+		StringBuilder out = new StringBuilder();
```

Comment:
My reasoning for using the default here, is that for any output less than BUFFER_SIZE, we end up over-allocating by specifying this value. If we use the default, and the first read fills the buffer, we will resize to BUFFER_SIZE immediately anyway when we write to the StringBuilder.